ember/app/templates/map/developments/development/index.hbs

```diff
@@ -445,11 +445,10 @@
         </p>
       </div>
 
-      {{#if hasDeletePermissions}}
-        <div class="actions">
-          <button class="styled negative" onclick={{action deleteDevelopment}}>Delete Development</button>
-        </div>
-      {{/if}}
+      <div class="actions">
+        {{#if hasDeletePermissions}}<button class="styled negative" onclick={{action deleteDevelopment}}>Delete Development</button>{{/if}}
+        {{#if session.isAuthenticated}}<button class="styled negative" onClick={{action flagDevelopment}}>Report Development for Review</button>{{/if}}
```

Comment:
This should say "Flag Development for Review" to keep with the rest of the language used.

Reply:
When signed in as someone with higher privileges than user, the two buttons have no margin between them. 
<img width="264" alt="capture d ecran 2018-05-01 a 12 31 55 pm" src="https://user-images.githubusercontent.com/6749126/39482110-bef9738e-4d3b-11e8-8717-bfb9a237c9bd.png">

Reply:
We should have an alternative state for boolean values. A user shouldn't be able to press the button more than once. You should put a placeholder here instead of a button when the development has been flagged.

We have two alternative states:

**Admin**: A button to "unflag" the development
**Non-Admin**: A static message letting the user know that the development has been flagged for review.